src/components/ModalAddToken.js

```diff
@@ -8,6 +8,7 @@
 import React from 'react';
 import $ from 'jquery';
 import tokens from '../utils/tokens';
```

Comment:
Missing import?

Reply:
Same as the other above

Reply:
I was a little confused, but it makes sense. I think it is ok.